jsettlers.graphics/src/jsettlers/graphics/map/draw/MapObjectDrawer.java

```diff
@@ -71,6 +72,10 @@
 	private static final int[] TREE_CHANGING_SEQUENCES = new int[] {
 			3, 3, 6, 9, 9, 18, 18,
 	};
+	static final float TREE_CUT_1 = 0.03F;
+	static final float TREE_CUT_2 = 0.06F;
+	static final float TREE_CUT_3 = 0.09F;
+	static final float TREE_TAKEN = 0.1F;
```

Comment:
I had a look at it. Logic really does not change this and is not actively in sync with the tree cutting process.

So this is good to be moved here.